man/man5/puppet.conf.5

```diff
@@ -344,6 +344,14 @@ The main Puppet configuration directory\. The default for this setting is calcul
 .
 .IP "" 0
 .
+.SS "facterng"
+Whether to enable a pre-Facter 4.0 release of Facter in ruby\. Defaults to false\.
+.
+.IP "\(bu" 4
+\fIDefault\fR: false
+.
+.IP "" 0
+.
```

Comment:
This file will be autogenerated on commit to puppet https://jenkins-master-prod-1.delivery.puppetlabs.net/view/puppet-agent/view/Acceptance%20Suites/view/master/view/Components/job/platform_puppet_vanagon-component-manpage-generation_master_agent-master/